test/expect-tests/action_tests.ml

```diff
@@ -1,42 +0,0 @@
-open Stdune
```

Comment:
Are these tests just gone? They seem pretty useful.

Reply:
(Although perhaps the corresponding code paths are already tested elsewhere.)

Reply:
They are indeed gone. They where testing the `Infer` pass, which has now been folded into the local `Action_expander` monad. I tried to test it by doing action expansion, but it is difficult because we need to construct an `Expander.t` in the code, which requires a context, scope db, etc...